.travis.yml

```diff
@@ -1,3 +1,5 @@
+sudo: required
+dist: trusty
```

Comment:
This shouldn't be required as Trusty is now the default environment used. See https://docs.travis-ci.com/user/precise-to-trusty-migration-guide/